src/rules/function-no-interpolation/README.md

```diff
@@ -0,0 +1,61 @@
+# function-no-interpolation
+
+The upcoming [first-class `calc()` feature](https://sass-lang.com/blog/request-for-comments-first-class-calc/) adds the ability to refer to variables and call functions directly within `calc()`, `clamp()`, `min()`, and `max()` expressions.
```

Comment:
This feature has now launched. It would also be good to connect this to the rationale for this lint as described in the issue